packages/channel-client/tests/fake-channel-client.test.ts

```diff
@@ -174,7 +174,7 @@ describe('FakeChannelClient', () => {
       // phase and the clients directly go to the channel 'closed' state
       const channelResult = await clientA.closeChannel(channelId);
       expect(channelResult).toEqual(states['closed']);
-      expect(providerB.latestState).toEqual(states['closed']);
+      expect(providerB.latestState[channelId]).toEqual(states['closed']);
     });
 
     it('player with invalid turn cannot make a valid close channel call', async () => {
```

Comment:
It would be nice to add a test to cover the case of multiple channels.

Reply:
Agreed. I'll raise an issue for that: I don't want to hold up progress on `web3torrent`.